### PR TITLE
ldap_exop

### PR DESCRIPTION
**Pull request by army1349**

### CODE REVIEW

**Comment by army1349** [comment]
Fixes #4992

File: plugins/password/drivers/ldap_exop.php [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+
+            ldap_unbind($ds);
+
+            return PASSWORD_CONNECT_ERROR;
+        }
+
+        $this->_debug("S: OK");
+
+        if (!function_exists('ldap_exop_passwd')) {
+            $this->_debug("ldap_exop_passwd not supported");
```
Comment (alecpl) [inline plugins/password/drivers/ldap_exop.php]: I would use rcube::raise_error() here and move the whole block to start of the save() method.

Comment (army1349) [reply plugins/password/drivers/ldap_exop.php]: Good idea, done.

**Comment by alecpl** [comment]
Merged. I de-duplicated some code, so please test after me.

**Comment by army1349** [comment]
There are few glitches, I have pushed fix to my fork.

**Comment by alecpl** [comment]
Create a new PR, please.

**Comment by army1349** [comment]
Sure, [PR](https://github.com/roundcube/roundcubemail/pull/6669) created.

**Pull request merged**